### ISSUE TITLE
updateBinaryGradWeight-factor

### ISSUE THREAD

**Issue by XinDongol** [jiecaoyu/XNOR-Net-PyTorch#20]
Thanks for your codes.
I have quick question about line 82 of 'CIFAR_10/util.py'. Why we have to 'mul(1.0-1.0/s[1]).mul(n)'?

**Comment by jiecaoyu** [comment]
@XinDongol Hi, from my understanding, ```mul(1.0-1.0/s[1])``` is the gradient for the ```meancenterConvParams()``` and ```mul(n)``` is used to increase the gradient to mitigate the effect of weight decay.

**Comment by XinDongol** [comment]
Thanks a lot. That makes sense.

**Comment by YanWang2014** [comment]
meancenterConvParams() is used in in_channels, do you know why? What about using out_channels?

**Comment by jiecaoyu** [comment]
@YanWang2014 Sorry but I am not sure about the reason. Probably a corresponding test will be helpful.

**Comment by YanWang2014** [comment]
Thank you
